docs/command_line_tool/alternate_install.md

```diff
@@ -23,3 +25,20 @@ Phylum is written in Rust, so you'll need a recent Rust installation to build it
    ```sh
    cargo install --path cli
    ```
+
+## Install with phylum-ci
+
+See the [phylum-ci](https://github.com/phylum-dev/phylum-ci) project for details.
```

Comment:
```suggestion
The [`phylum` Python package](https://pypi.org/project/phylum/) provides a script entry point, `phylum-init`, for bootstrapping the Phylum CLI.
It requires a modern version of Python. See the [phylum-ci](https://github.com/phylum-dev/phylum-ci) project for full detail.
```

Reply:
> It requires a modern version of Python.

I'm rejecting this note because Python 2.x has been end-of-life for longer than Phylum has been a company.